pkg/bootstrapper/cloud/cloud.go

```diff
@@ -0,0 +1,25 @@
+package cloud
+
+import (
+	"strings"
+)
+
+const (
+	awsPlatformType = "aws"
+)
+
+// GetOverriddenKubeletHostname returns correct hostname for kubelet if it should
+// be overridden, or an empty string otherwise.
+func GetOverriddenKubeletHostname(platformType string) (string, error) {
```

Comment:
Nit: I'd probably call this `GetKubeletHostnameOverride` so that it looks more like the kubelet flag `hostname-override`, but I'll defer to WMCO folks as to whether they think that's a required change

Reply:
Done